docs/en/notes/finetune_custom_dataset.md

```diff
@@ -0,0 +1,317 @@
+```
+
+```{tip}
+Refers to [Learn about Configs](../user_guides/config.md) for more detailed configurations.
+```
+
+## Start Training
+
+Now, we have finished the fine-tuning config file as following:
+
+```python
+_base_ = [
+    '../_base_/models/resnet50.py',           # model settings
+    '../_base_/datasets/imagenet_bs32.py',    # data settings
+    '../_base_/schedules/imagenet_bs256.py',  # schedule settings
+    '../_base_/default_runtime.py',           # runtime settings
+]
+
+# model settings
+model = dict(
+    backbone=dict(
+        frozen_stages=2,
+        init_cfg=dict(
+            type='Pretrained',
+            checkpoint='https://download.openmmlab.com/mmclassification/v0/resnet/resnet50_8xb32_in1k_20210831-ea4938fc.pth',
+            prefix='backbone',
+        )),
+    head=dict(num_classes=10),
+)
+
+# data settings
+data_root = 'data/custom_dataset'
+train_dataloader = dict(
+    dataset=dict(
+        type='CustomDataset',
+        data_root=data_root,
+        ann_file='',       # We assume you are using the sub-folder format without ann_file
+        data_prefix='train',
+    ))
+val_dataloader = dict(
+    dataset=dict(
+        type='CustomDataset',
+        data_root=data_root,
+        ann_file='',       # We assume you are using the sub-folder format without ann_file
+        data_prefix='test',
+    ))
+test_dataloader = val_dataloader
+
+# schedule settings
+optim_wrapper = dict(
+    optimizer=dict(type='SGD', lr=0.01, momentum=0.9, weight_decay=0.0001))
+param_scheduler = dict(
+    type='MultiStepLR', by_epoch=True, milestones=[15], gamma=0.1)
+```
+
+Here we use 8 GPUs on your computer to train the model with the following command:
+
+```shell
+bash tools/dist_train.sh configs/tutorial/resnet50_finetune_cifar.py 8
+```
+
+Also, you can use only one GPU to train the model with the following command:
+
+```shell
+python tools/train.py configs/tutorial/resnet50_finetune_cifar.py
+```
+
+But wait, an important config need to be changed if using one GPU. We need to
+change the dataset config as following:
+
+```python
+data_root = 'data/custom_dataset'
+train_dataloader = dict(
+    batch_size=128
+    dataset=dict(
+        type='CustomDataset',
+        data_root=data_root,
+        ann_file='',       # We assume you are using the sub-folder format without ann_file
+        data_prefix='train',
+    ))
+val_dataloader = dict(
+    dataset=dict(
+        type='CustomDataset',
+        data_root=data_root,
+        ann_file='',       # We assume you are using the sub-folder format without ann_file
+        data_prefix='test',
+    ))
+test_dataloader = val_dataloader
+```
+
+It's because our training schedule is for a batch size of 128. If using 8 GPUs,
+just use `batch_size=16` config in the base config file for every GPU, and the total batch
+size will be 128. But if using one GPU, you need to change it to 128 manually to
+match the training schedule.
```

Comment:
Besides, I think we could introduce the usage of `auto_scale_lr` here. With `auto_scale_lr`, we don't need to change the batch_size?